### PR TITLE
Button overlay

### PR DESCRIPTION
**Pull request by Eyeveri**
Adds accented buttons to the Suit jacket, blazer and long blazer, to add more depth to the loadout option(s).

### CODE REVIEW

**Comment by Eyeveri** [comment]
!wip

**Comment by Eyeveri** [comment]
There is a small bug that @1Sparky1 will have to help me with, before this can be merged/finished. But the work is 90% done.

**Review by Generalcamo — CHANGES_REQUESTED** [review]
You should remove the old, now outdated icons, rather than keeping orphaned unused icons in the files.

**Comment by Eyeveri** [comment]
> 

I don't follow, the only icons being used are the ones that are being overlayed. Wezzy and I manually built the file for it.

**Comment by Eyeveri** [comment]
@1Sparky1 fixed the bug! Yay!

!review

File: html/changelogs/EyeveriChangelog.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Adds"
```
Comment (Generalcamo) [inline html/changelogs/EyeveriChangelog.yml:41]: You missed this though.

**Review by Generalcamo — APPROVED** [review]
Code looks fine, old icons are gone.

File: html/changelogs/EyeveriChangelog.yml (hunk 1 above)
Comment (Eyeveri) [reply html/changelogs/EyeveriChangelog.yml:41]: Yeahhhh... Just saw that lmao

Comment (Eyeveri) [reply html/changelogs/EyeveriChangelog.yml]: I got a little excited. Pushed the fix!

**Review by NonQueueingMatt — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**